WalletWasabi/Legal/Assets/LegalDocuments.txt

```diff
@@ -258,7 +258,7 @@ COMPLIANCE WITH TAX OBLIGATIONS
 COUNTRY OF RESIDENCE
 ====================
 
-    In case if You are a Gibraltar resident and willing to use Wasabi Wallet please inform us in advance on the following e-mail address: legal@zksnacks.com.
+    In case You are a Gibraltar resident and willing to use Wasabi Wallet please inform us in advance on the following e-mail address: legal@zksnacks.com.
```

Comment:
@bharmat

Reply:
This PR is fixing grammar which is good but I would avoid to change grammar in C# code and in the license text at the same time in one PR in the future. The consequences of changing legal texts are far reaching and should be discussed separately, imho.

Reply:
Wait for @bharmat answer's